integration_tests/tests/test_telemetry/run.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
```

Comment:
This was a surprise that emerged from my commit squashing. I'm not clear what git sorcery caused it.